### ISSUE TITLE
Excalibur sandbox test compilation fails locally

### ISSUE THREAD

**Issue by jedeen** [excaliburjs/Excalibur#1010]
Currently, the build and test scripts do not appropriately clear out files from previous runs for the sandbox tests. We need to update our build automation to account for this.

### Steps to Reproduce
`npm run all`

### Expected Result
Successful build of Excalibur and run of the test suite

### Actual Result
Compilation fails  for the sandbox tests

### Current Workaround
Delete the compiled files in the /sandbox folder

**Comment by alanag13** [comment]
I experienced this problem too, but once I deleted the compiled files it didn't happen anymore. I think we must have changed something about the way the visual tests compile/where the scripts come from because new builds do not create compiled files in the /sandbox folder, but we can still successfully run the sandbox after they are removed. Best I can tell, anyone cloning the repo fresh shouldn't have this problem.
